src/components/common/styles/Resources.css

```diff
@@ -32,3 +32,43 @@
 .ant-form-item-label label {
   font-size: 16px;
 }
+
+          /* RESOURCE FORM */
+
+/* resource/add mentor review form headers */
+.ant-card .ant-card-head { 
+    background: #003d71;
+}
+/* resource header options */
+.ant-tabs-tab-btn {
+    color: rgb(255, 255, 255, 0.85) !important;
+}  
+/* resource header bar */
+.ant-tabs-ink-bar-animated {
+    background: rgb(255, 255, 255, 0.85) !important;
+    position: relative;
+    top: 45px;
+}
+/* resource form titles */
+.ant-form-horizontal .ant-form-item label {
```

Comment:
this pseudo-code is a nice way to translate the ant design titles--thanks for these.